tests/lms/views/error_test.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from pyramid import httpexceptions
+import pytest
+
+from lms.views import error
+from lms.exceptions import MissingLtiLaunchParamError
+
+
+class TestErrorController(object):
```

Comment:
We actually don't need `(object)` on classes in the LMS app because it's a Python 3 app. Just `class TestErrorController:`